scripts/pythonContractConverter.py

```diff
@@ -0,0 +1,115 @@
+# assumes all return types are strings
+def parseContract(file, funcName):
+    import re
+
+    fin = -1
+    try:
+        fin = open(file)
+    except:
+        print("[ERROR] {fname} cannot be opened".format(fname = file))
+        return ""
+
+    data = fin.read()
+    fin.close()
```

Comment:
The with handles file closing automatically
```suggestion
    data = ''
    try:
        with open(file, 'r') as fi:
            data = fi.read()
    except:
        print(f"[ERROR] {file} cannot be opened")
        return ""
```